com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1855,11 +1854,11 @@ public void setTextureProfiles(TextureProfiles textureProfiles) {
     }
 
     public void excludeCollectionProxy(String path) {
-        this.excludedCollectionProxies.add(path);
+        state.addExcludedCollectionProxy(path);
```

Comment:
We know if collection should be excluded or not from `CollectionProxyBuilder` which we run only once and then save its signature in `state` file. 
Next time we run archiving process - all the signatures are equal and we skip `CollectionProxyBuilder` step, so we don't know which collections should be excluded.

To avoid full rebuilding I keep this info as part of the cache state.